### ISSUE TITLE
Unabled to create GCP upgrade cluster within time

### ISSUE THREAD

**Issue by terrcin** [opstrace/opstrace#1215]
No obvious errors, just didn't create the cluster quick enough.

https://buildkite.com/opstrace/scheduled-main-builds/builds/2815#f2eff76c-d730-49a6-92ce-68abe645c3d4/6-1906

```
[2021-08-08T06:28:55Z] 2021-08-08T06:28:55.323Z info: Closing clusterChannel and shutting down informers
--
  | [2021-08-08T06:28:55Z] 2021-08-08T06:28:55.323Z warning: cluster teardown attempt timed out after 1200 seconds
  | [2021-08-08T06:28:55Z] 2021-08-08T06:28:55.324Z debug: do not log error detail
  | [2021-08-08T06:28:55Z] 2021-08-08T06:28:55.324Z error: 5 attempt(s) failed. Stop retrying. Exit.
  | [2021-08-08T06:28:55Z] 2021-08-08T06:28:55.324Z debug: shut down logger, then exit with code 1
  | [2021-08-08T06:28:55Z] make: *** [Makefile:725: testupgrade-teardown] Error 1
  | [2021-08-08T06:28:55Z] make: *** [ci-testupgrade-teardown] Error 2
  | [2021-08-08T06:28:55Z] 🚨 Error: The command exited with status 2
  | [2021-08-08T06:28:55Z] user command error: exit status 2
```

**Comment by terrcin** [comment]
https://buildkite.com/opstrace/scheduled-main-builds/builds/2813#b09116e8-7d2c-47f8-b9a5-337595b4202a/6-1908

**Comment by jgehrcke** [comment]
> No obvious errors, just didn't create the cluster quick enough.

critical error detail here is that this timed out during the "waiting for deployments" phase, practically waiting for all of those.  likely this was actually #1217  or related to that.

**Comment by jgehrcke** [comment]
@sreis can you maybe narrow down the problem that happened here, and then adjust the title of this ticket / maybe merge with a ticket that already exists?

**Comment by sreis** [comment]
```
[2021-08-08T04:40:30Z] 2021-08-08T04:40:30.950Z [32minfo[39m: controller image is already at desired version: opstrace/controller:83d48f59-ci
```

Closing. It's a duplicate of #1217 and fixed with #1223.
